apps/pdftk/.env.docker

```
@@ -1,2 +1,2 @@
 PDFTK_VERSION=3.3.3
-JRE_VERSION=2:1.17-74
+JRE_VERSION=11.0.27_6
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The JRE_VERSION key should go before the PDFTK_VERSION key

(UnorderedKey)